### ISSUE TITLE
Clicking link in reliablity monitor generated deflector crash report.

### ISSUE THREAD

**Issue by pdhixenbaugh** [spikespaz/search-deflector#81]
> **IMPORTANT:** READ EVERYTHING!!! That includes the wiki. Make sure you've skimmed the wiki before making a bug report.
> https://github.com/spikespaz/search-deflector/wiki
>
> This issue should be properly formatted in markdown. If you don't know what that is, read the following website before continuing:
> https://guides.github.com/features/mastering-markdown/
>
> Most of the text in this issue should be replaced with your own or deleted. Do not delete section headers unless the information they would contain is irrelevant or inapplicable.

## Bug Description

If you have any other details, replace this line, otherwise delete it.

**File:** `source\deflector.d`

**Line:** 53

**Message:**

```
There was an error deflecting your search. Passed URI is:	microsoft-edge:https://go.microsoft.com/fwlink/?LinkId=517009
```

## How to Reproduce

**Query string:**
```

```

**Query or URL:**
```

```

**Steps to reproduce the behavior:**

## Expected Behavior

I clicked on a link in Reliability Monitor about how to get more information on a crash; I expected to be taken to a website.

## Environmental Details

**Browser**: UNKNOWN STRING

**Search Engine:** UNKNOWN STRING

**Custom browser path:**
```

```

**Custom search engine URL:**
```
duckduckgo.com/?q={{query}}
```

| Windows Version | `1909` |
| --- | --- |
| Build Number | `18363` |
| Edition | `Enterprise` |
| Insiders Preview |  |

**Comment by pdhixenbaugh** [comment]
EDIT: I'm not sure how much of Search Deflector is currently blocked due to Windows Antivirus...

**Comment by spikespaz** [comment]
This is fixed by 49e833f6eae382acd872889709954d28ca441b90. The changes will be available in the new version.

As for your second statement, if Search Deflector works at all nothing is blocked by Defender.
